### PR TITLE
Add suffix to compose project names

### PR DESCRIPTION
**Pull request by mrodm**
Follows #1838

Add suffix (run ID string) to compose project created by terraform and custom agent service deployers.

### CODE REVIEW

File: internal/servicedeployer/custom_agent.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func (d *CustomAgentDeployer) SetUp(ctx context.Context, svcInfo ServiceInfo) (D
 
 	service := dockerComposeDeployedService{
 		ymlPaths: ymlPaths,
-		project:  "elastic-package-service",
+		project:  fmt.Sprintf("elastic-package-service-%s", svcInfo.Test.RunID),
```
Comment (mrodm) [inline internal/servicedeployer/custom_agent.go:109]: I've updated this service deployer to keep consistency.
But it would not be needed, since this deployer is not used with independent Elastic Agents. 

Would it be better to revert the change in this file?

This would make mandatory to change test configuration files for `oracle` package using the placeholders `{{Hostname}}` and `{{Port}}`.

Comment (jsoriano) [reply internal/servicedeployer/custom_agent.go:109]: Well, it doesn't do any harm, right? Keep the change in case this continues to be used.

**Review by jsoriano — APPROVED** [review]
